### ISSUE TITLE
Wrong info for {{ownerName}} and {{repoName}} when Repo name has '.' in it

### ISSUE THREAD

**Issue by bbendes** [jenkinsci/git-changelog-plugin#24]
Some of my repos have a period '.' in the name; instead of return the correct {{owner}} and {{repoName}} it returns the value before the '.' as the owner and the value after the '.' as the repoName.

 * Plugin: 2.7
 * Jenkins: 2.137
 * Your configuration:
   * Github link: git@github.Company.com:Organization/5.0.git
   * Jenkins File:
```
def changelogString = gitChangelog from: [type: 'REF', value: "${Previous_Version}"], to: [type: 'REF', value: "${version}, "repo: ".\\${Application}", returnType: 'STRING', template: '''
# Changelog for {{ownerName}} {{repoName}}.
''']

currentBuild.description = changelogString
```
 * Expected result and actual result.
   * Result: "# Changelog for 5 0."
   * Expected: "# Changelog for Organization 5.0."

**Comment by tomasbjerre** [comment]
https://github.com/tomasbjerre/git-changelog-lib/commit/56f23fe9eff5487

**Comment by tomasbjerre** [comment]
Should be fixed now in 2.8. Open issue again if not working!
